src/cli.js

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env node
-var caseCoverage = require('./index.js');
+
 var [ , , ... args ] = process.argv;
-var configPath = (args[0])? args[0] : './spec/fixtures/case-coverage.json';
-caseCoverage.run(configPath)
-.catch(function (error) {
-    console.log(error);
-});
+var configPath       = args[0] || './spec/fixtures/case-coverage.json';
+var caseCoverage     = require('./index.js');
+
+caseCoverage
+    .run(configPath)
+    .catch(console.log);
```

Comment:
@cookiescrumbs I think you can pass console.log directly here as a function reference because .catch takes a function with one argument, the error, so it might as well be console.log instead of a function with one argument that calls console.log?